From 8b5ad2dbc102e95281168e02dce5f34680175e7f Mon Sep 17 00:00:00 2001
From: Yichao Yu <yyc1992@gmail.com>
Date: Fri, 10 Jun 2022 08:31:10 -0400
Subject: [PATCH 4/5] Change default BUILDTOOL to devtools-archlinuxcn

While allowing settings it back to devtools. See discussions on [1].

[1] https://github.com/archlinuxcn/repo/issues/2798
---
 archbuild.in  | 2 +-
 lib/common.sh | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/archbuild.in b/archbuild.in
index e6cf19a..64f5b12 100644
--- a/archbuild.in
+++ b/archbuild.in
@@ -54,7 +54,7 @@ while getopts 'hcr:' arg; do
 	esac
 done
 
-check_root SOURCE_DATE_EPOCH,SRCDEST,SRCPKGDEST,PKGDEST,LOGDEST,MAKEFLAGS,PACKAGER,GNUPGHOME
+check_root SOURCE_DATE_EPOCH,SRCDEST,SRCPKGDEST,PKGDEST,LOGDEST,MAKEFLAGS,PACKAGER,GNUPGHOME,BUILDTOOL
 
 # Pass all arguments after -- right to makepkg
 makechrootpkg_args+=("${@:$OPTIND}")
diff --git a/lib/common.sh b/lib/common.sh
index 0996247..99bef11 100644
--- a/lib/common.sh
+++ b/lib/common.sh
@@ -16,7 +16,7 @@ $_INCLUDE_COMMON_SH
 export LANG=C
 
 # Set buildtool properties
-export BUILDTOOL=devtools
+export BUILDTOOL=${BUILDTOOL:-devtools-archlinuxcn}
 export BUILDTOOLVER=m4_devtools_version
 
 shopt -s extglob
-- 
2.36.1

